### DISCUSSION TITLE
有QQ群戳一戳功能吗？在文档里没看到相关的介绍。

### DISCUSSION THREAD

**Discussion by ikaros143** [simple-robot/Discussions#98] [Q&A] [answered]
bot接收戳一戳或者戳一戳其他群员这种的。

**Comment by ForteScarlet** [comment, marked as answer]
参考
- [MiraiNudge](https://docs.simbot.forte.love/components/mirai/simbot-component-mirai-core/love.forte.simbot.component.mirai.message/-mirai-nudge/index.html)
- [MiraiNudgeEvent](https://docs.simbot.forte.love/components/mirai/simbot-component-mirai-core/love.forte.simbot.component.mirai.event/-mirai-nudge-event/index.html)

> Note: 疑问请选择 `Q&A` 分类

**Reply by ikaros143** [reply]
非常感谢^_^
